doc/source/user_guide/ug_case_settings.rst

```diff
@@ -472,14 +476,14 @@ zbgc_nml
    "``K_Sil_phaeo``", "real", "silicate half saturation phaeocystis mmol/m^3", "0.0"
    "``K_Sil_sp``", "real", "silicate half saturation small plankton mmol/m^3", "0.0"
    "``kn_bac_protein``", "real", "bacterial degradation of DON per day", "0.2"
-   "``l_sk``", "real", "characteristic diffusive scale in m", "20.0"
+   "``l_sk``", "real", "characteristic diffusive scale in m", "2.0"
```

Comment:
The default in Icepack and in namelist is currently 20 and we use 20.  Should it be 2?

Reply:
yes.